.changeset/silver-eagles-reply.md

```diff
@@ -0,0 +1,23 @@
+---
+'@backstage/create-app': patch
+---
+
+Permission the catalog-import route
```

Comment:
```suggestion
Permission the `catalog-import` route
```

FYI accepting suggestions in the UI requires that your signed-of-by line is added to the commit message :)